scripts/wrappers/reset.py

```diff
@@ -269,7 +269,8 @@ def preflight_check():
 )
 def reset(destroy_storage):
     """
-    Main reset flow
+    TODO: add proper docstring/help message
+    Main reset  flow
```

Comment:
Any suggestions on proper message here? I was thinking of adding a summary of this https://microk8s.io/docs/command-reference#heading--microk8s-reset